zap/src/main/resources/org/zaproxy/zap/resources/config.xml

```diff
@@ -123,6 +123,56 @@
 				<resBodyRegex/>
 				<enabled>true</enabled>
 			</scanner>
+			<scanner>
+				<name>json_extended</name>
+				<type>TAG</type>
+				<config>JSON</config>
+				<reqUrlRegex/>
+				<reqHeadRegex/>
+				<resHeadRegex>content-type:\s{0,10}.{1,20}\/.{0,100}json</resHeadRegex>
+				<resBodyRegex/>
+				<enabled>false</enabled>
+			</scanner>
+			<scanner>
+				<name>response_yaml</name>
+				<type>TAG</type>
+				<config>YAML</config>
+				<reqUrlRegex/>
+				<reqHeadRegex/>
+				<resHeadRegex>content-type:\s{0,10}(?:text\/yaml|application\/yaml)</resHeadRegex>
+				<resBodyRegex/>
+				<enabled>false</enabled>
+			</scanner>
+			<scanner>
+				<name>yaml_extended</name>
```

Comment:
Do we really need these? Or, why not just these since they already include the others.

Reply:
That'd be fine with me for XML and YAML.

Reply:
Should be fine for JSON too, since these just affect new installations. On upgrade you could add the extended though.

Reply:
I thought you'd previously said we shouldn't change the existing pattern in case someone was depending on it somehow.